packages/commons/src/email-manager/emailManager.ts

```diff
@@ -67,7 +69,11 @@ export function initSesMailManager(awsConfig: AWSSesConfig): EmailManagerSES {
 
   return {
     kind: "SES",
-    send: async (mailOptions: Mail.Options): Promise<void> => {
+    send: async (
+      mailOptions: Mail.Options,
+      logger: Logger,
+      skipError: boolean = false
```

Comment:
Agree with both points, and also, I would name it `skipTooManyRequestsError` to be more specific, since at the moment we don't skip any other error